R/language.R

```diff
@@ -21,36 +22,115 @@
 #' is_lang(q3)
 #' is_call(q3)
 #'
-#' # A literal is either a scalar atomic vector or a name:
-#' is_literal("string")
-#' is_literal(letters)
-#' is_literal(quote(name))
+#'
+#' # Atomic language objects are the terminating nodes of a call
+#' # tree. They are either a scalar atomic vector, a name, or NULL:
+#' is_atomic_lang("string")
```

Comment:
Yeah, agreed.  

It's a literal in the sense that it's a leaf node in the parse tree.  Using code you can manipulate the tree to contain any R object, but you can't get there from parsing text.